### ISSUE TITLE
Why new framework?

### ISSUE THREAD

**Issue by j209** [konsoletyper/teavm-flavour#25]
Hi
Would not it be better to work on existing frameworks like [Craete-React-Kotlin-App](https://github.com/JetBrains/create-react-kotlin-app) instead of creating a new framework for java-based front-end?

**Comment by konsoletyper** [comment]
Why people keep creating frameworks while there're plenty of existing ones? Why did people create React while there were jQuery, Dojo, Knockout, Backbone, etc?

In my case the reasons are:
1. Kotlin team was not interested in incorporating TeaVM support in their framework
2. Personally, I think that React is built on concepts that don't fit with Java/Kotlin/Scala idioms. You end up with tons of wrappers (and still keep dealing with leaky abstractions). Why do you think Google wrote separate Angular implementation for Dart instead of bingind to Typescript implementation?
3. I started Flavour before Kotlin was even released.

**Comment by cyberquarks** [comment]
Yes I think what Kotlin have now is Ktor, to me it looks like Vert.x 

@j209 TeaVM/Flavour at first looks unusual but the more you use it, considering it TeaVM at core, it makes more sense. At first I was skeptical about why and how it works comparing to other framework; but don't compare it to other frameworks, Flavour is a new kind of beast with a different punch/approach which as we use it I find allows as to make webapps super fast.
